launchable/commands/test_path_writer.py

```diff
@@ -7,7 +7,7 @@
 
 
 class TestPathWriter(object):
-    base_path = Optional[str]
+    base_path = None  # type: Optional[str]
```

Comment:
I don't know why this error happened on this PR but fixed to pass the CI

https://github.com/launchableinc/cli/actions/runs/4120619904/jobs/7115523912
https://github.com/launchableinc/cli/actions/runs/4121101420/jobs/7116471305

Reply:
That's because we use Python 3.5 which does not support type hint in variable.